packages/client/libs/typed-event-emitter/index.ts

```diff
@@ -41,4 +41,20 @@ export class TypedEventEmitter<
 
 		return this;
 	}
+
+	/**
+	 * This creates a promise that you can use for a single event.
+	 * @param event A event name
+	 */
+	public promise<E2 extends E, P extends unknown[] = Parameters<T[E2]>>(
```

Comment:
```suggestion
	public promise<E2 extends E, P = Parameters<T[E2]>>(
```

I think that this is more simple

Reply:
Thanks I agree.